### PR TITLE
Added Quickfix List plugin

### PR DESCRIPTION
**Pull request by cibinmathew**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [x] If my package is a syntax it doesn't also add a color scheme. ***
- [x] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My package is an alternative for vim's quickfix list navigation

There are no packages like it in Package Control.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by braver** [comment]
You might want to try and re-submit this. I'm not sure what's wrong with the bot.

**Comment by cibinmathew** [comment]
done. @braver please retrigger the bot

**Comment by braver** [comment]
Not sure what's wrong with the bot. .. we can try to merge it and solve issues (of there are any) afterwards, the submission looks fine to me.

One thing about the package though:
- settings keys are usually in lowercase
- is the settings file path an absolute path? I would normally put such file in `~/` in a dot file. Or even better, put it in the Packages/User directory since your tool lives in Sublime Text

**Comment by cibinmathew** [comment]
> Not sure what's wrong with the bot. .. we can try to merge it and solve issues (of there are any) afterwards, the submission looks fine to me.
> 
> One thing about the package though:
> 
> * settings keys are usually in lowercase
> * is the settings file path an absolute path? I would normally put such file in `~/` in a dot file. Or even better, put it in the Packages/User directory since your tool lives in Sublime Text

@braver Feedbacks implemented in plugin repo's new release

File: repository/q.json [hunk 1]
```diff
@@ -239,6 +239,19 @@
...
+		{
+			"name": "Quickfix List",
+			"details": "https://github.com/cibinmathew/sublime_quickfix_list",
+			"author": "Cibin Mathew",
+			"labels": ["navigation"],
+			"releases": [
+				{
+					"sublime_text": "*",
+					"platforms": ["osx", "linux"],
+					"branch": "master"
```
Comment (braver) [inline repository/q.json:251]: Oh, wait, you need to remove the branch here and set it to `"tags": true`. We don't support branch based releases anymore, and you are properly tagging releases already.

Comment (braver) [reply repository/q.json:251]: That's maybe also why the bot is failing.

Comment (cibinmathew) [reply repository/q.json]: @braver I have made the mentioned changes

**Pull request merged**